### DISCUSSION TITLE
Enabling date and reading time for posts

### DISCUSSION THREAD

**Discussion by spindev** [lxndrblz/anatole#421] [Q&A] [answered]
Hi @lxndrblz, nice theme. I adopted it for my blog and wanted to show the date and the reading time below the header but it doesn't show up. What config setting did I miss?

<img width="696" alt="image" src="https://user-images.githubusercontent.com/14042276/212431712-0f8a9ab0-6a10-4a7e-8de6-6c806827f005.png">

Thanks in advance :)

**Comment by lxndrblz** [comment, marked as answer]
@spindev You would either need to set the type to post or define a custom `.Site.Params.postSectionName`. In your case the postSectionName would be posts (mind the s).

**Reply by spindev** [reply]
Thx. Fixed it with changing the type to post. Was too late yesterday at all 🙈 Have a nice weekend :)
